DC_TTS_test.ipynb

```diff
@@ -0,0 +1,244 @@
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "H7nbaFrOfZtJ"
+      },
+      "source": [
+        "# DC_TTS - a test on the pre-trained model which is based on the LJ speech dataset"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9rYinl7keOQx"
+      },
+      "source": [
+        "**!!** **Remember to activate Hardware acceleration** **!!**\n",
+        "Edit → Notebook Settings → select GPU from the hardware accelerator menu\n",
+        "\n",
+        "This notebook allows you to synthesize English sentences with a pre-trained model based on the LJ speech dataset.\n",
+        "After doing some preliminary steps, you will be able to choose your own sentences that should be synthesized or stick with the sentences pre-defined in the [harvard_sentences](http://www.cs.columbia.edu/~hgs/audio/harvard.html).txt. At the end you will be able to use a simple audio widget to play the synthesized files."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "jQunD3-twV2A"
+      },
+      "source": [
+        "# clone the repository\n",
+        "!git clone https://github.com/jkuhlemann/dc_tts.git"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "ZNFKOgHQOtGL"
+      },
+      "source": [
+        "# create folders necessary for everything to work\n",
+        "!mkdir /content/dc_tts/samples\n",
+        "!mkdir /content/dc_tts/logdir\n",
+        "!mkdir /data/\n",
+        "!mkdir /data/private/\n",
+        "!mkdir /data/private/voice/"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "RhQIjHVdvm6U"
+      },
+      "source": [
+        "# before going further, we have to change some lines in\n",
+        "# hyperparams.py, so that it loads the right directories\n",
+        "import fileinput\n",
+        "import sys\n",
+        "\n",
+        "def replacement(file, previousw, nextw):\n",
+        "   for line in fileinput.input(file, inplace=1):\n",
+        "       line = line.replace(previousw, nextw)\n",
+        "       sys.stdout.write(line)\n",
+        "orig_1 = \"test_data = 'harvard_sentences.txt'\"\n",
+        "repl_1 = \"test_data = '/content/dc_tts/harvard_sentences.txt'\"\n",
+        "\n",
+        "orig_2 = 'logdir = \"logdir/LJ01\"'\n",
+        "repl_2 = 'logdir = \"/content/dc_tts/logdir/LJ01\"'\n",
+        "\n",
+        "orig_3 = \"sampledir = 'samples'\"\n",
+        "repl_3 = \"sampledir = '/content/dc_tts/samples'\"\n",
+        "file = \"/content/dc_tts/hyperparams.py\"\n",
```

Comment:
It is better practice to create the variables in the order of using them. So in this case it would be neater if "file = ..." was created before "orig_1 = ...", because when you call the function, the variables occur in the following order: "replacement(file, orig_1, ...)"

Reply:
changed that!